ts_scripts/sanity_utils.py

```diff
@@ -163,51 +162,18 @@ def run_rest_test(model, register_model=True, unregister_model=True):
 
 
 def test_sanity():
-    generate_grpc_client_stubs()
```

Comment:
I guess sanity test will fail without gRPC client.

Reply:
Thats call is now a fixture https://github.com/pytorch/serve/blob/15a9fd9abd703a0291a87a80b46fe06c538373aa/test/pytest/sanity/test_model_registering.py#L15